.github/workflows/test-npm.yml

```diff
@@ -0,0 +1,24 @@
+name: Integration Test npm
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        container:
+        - node:12
```

Comment:
May be worth expanding out the matrix to cover supported versions or at least a version with NPM 6 vs NPM 7

Reply:
@nschonni makese sense (although probably good for a separate PR), do you have a recommendation?